bin/crew

```diff
@@ -795,13 +797,16 @@ def pre_install(dest_dir)
 end
 
 def post_install
-  if @pkg.print_source_bashrc?
+  if @pkg.print_source_bashrc? || @pkg.gnome?
     ExitMessage.add <<~PRINT_SOURCE_BASHRC_EOT.lightblue, print_last: true
 
       To finish the installation, please execute the following:
       source ~/.bashrc
     PRINT_SOURCE_BASHRC_EOT
   end
+
+  GnomePostinstall.activate if @pkg.gnome?
```

Comment:
Instead of calling a method, just set the flag here.  `@gnome_postinstall = true if @pkg.gnome?`